### DISCUSSION TITLE
Show loading to remove products in the menu cart widget

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#21172] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

No preload is displayed when users remove products in the menu cart widget. The lack of this feature makes users think that the desired product is not being removed.

### Describe the solution you'd like

In the menu cart widget, when the delete product button is clicked, first a loading is displayed and then the product is deleted.
Please watch the video below. When the remove product button is clicked, a loading is displayed on the product image and then the product is removed.

https://user-images.githubusercontent.com/77163248/209560901-5695a638-46fe-4cb2-8232-662f9374b6dc.mp4

**Comment by Vossha** [comment]
+1

**Comment by nicholaszein** [comment]
Hello @79ho3ein,

thanks for submitting your feature request! 🙏 

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap for the time being. 

🔖 We will, of course, keep this suggestion in mind and consider adding it if there is significant demand in the future.

Best regards
